src/main/java/com/icthh/xm/tmf/ms/resource/web/rest/errors/ExceptionTranslator.java

```diff
@@ -19,14 +13,22 @@
 import org.zalando.problem.ProblemBuilder;
 import org.zalando.problem.Status;
 import org.zalando.problem.spring.web.advice.ProblemHandling;
+import org.zalando.problem.spring.web.advice.security.SecurityAdviceTrait;
 import org.zalando.problem.violations.ConstraintViolationProblem;
 
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version